Client/Frontend/Browser/TabManager.swift

```diff
@@ -1145,6 +1145,29 @@ class TabManagerNavDelegate: NSObject, WKNavigationDelegate {
           pref = preference
         })
     }
+    
+    if res != .cancel {
+      // Mixed-Content validation
+      if let url = navigationAction.request.url,
```

Comment:
is this going to show an alert on all https ->

Reply:
Fixed to block only `form` requests.